content/cloud/changelog/cloud-custom-domains.md

```diff
@@ -8,7 +8,7 @@ type= "changelog_post"
 
 ---
 
-Fermyon Cloud is excited to share that it now supports custom domains. Until today, Spin applications were limited to either default randomly generated domain names or custom Fermyon subdomains; however, we heard your feedback that custom domains were an important feature to bring blogs and personalized websites to Fermyon Cloud. Using a custom domain name with your Spin application can improve your workload's SEO, build brand integrity, and improve discoverability. We encourage developers to bring their custom domains to Fermyon Cloud and assign them to their favorite Spin applications. Slat's has already gotten a headstart at [finickywhiskers.com](https://www.finickywhiskers.com/index.html). Let us know your thoughts on the feature at [feedback@fermyon.com](mailto:feedback@fermyon.com). 
+Fermyon Cloud is excited to share that it now supports custom domains. Until today, Spin applications were limited to default randomly generated domain names or custom Fermyon subdomains; however, we heard your feedback that custom domains were essential in bringing blogs and personalized websites to Fermyon Cloud. Using a custom domain name with your Spin application can improve your workload's SEO, build brand integrity, and improve discoverability. We encourage developers to bring their custom domains to Fermyon Cloud and assign them to their favorite Spin applications. Slat's has already gotten a headstart at [finickywhiskers.com](https://finickywhiskersdiary.com). Please let us know what you think about the feature at [feedback@fermyon.com](mailto:feedback@fermyon.com). 
```

Comment:
I can’t do a suggestion on this device but:

* the character’s name is Slats not Slat’s
* The diary link text still shows the game URL

Reply:
Thanks @itowlson